src/pages/ethaku.svelte

```diff
@@ -330,28 +466,37 @@
           </div>
         </div>
         <div
-          class="w-full mx-auto center flex flex-col md:flex-row items-center">
+          class="tarkennettu-row w-full mx-auto center flex flex-col md:flex-row items-center">
           <span
-            class="w-full md:w-1/2 uppercase text-ashblue tracking-widest font-bold">
-            Lämmitetty Nettoala (M2)
+            class="tarkennettu-label w-full md:w-1/2 text-ashblue tracking-widest">
+            {$_('ETHAKU_LAMMITETTY_NETTOALA')}
           </span>
           <div
             class="w-full md:w-1/2 flex justify-between items-center text-center">
             <div class="w-2/5">
-              <InputText label={'?'} />
+              <InputNumber
+                label={''}
+                min="0"
+                max={nettoalaMax}
+                bind:value={nettoalaMin} />
             </div>
-            <span class="material-icons"> horizontal_rule </span>
+            <span class="material-icons text-darkgrey"> horizontal_rule </span>
             <div class="w-2/5">
-              <InputText label={'?'} />
+              <InputNumber
+                label={''}
+                min={nettoalaMin}
+                bind:value={nettoalaMax} />
             </div>
           </div>
         </div>
       </div>
     {/if}
 
     <div class="w-full md:w-11/12 mt-4 flex flex-col sm:flex-row">
-      <Button {...buttonStyles.green}>Hae</Button>
-      <Button {...buttonStyles.ashblue}>Tyhjennä hakuehdot</Button>
+      <Button {...buttonStyles.green} on:click={submitForm}>{$_('HAE')}</Button>
+      <Button {...buttonStyles.ashblue} on:click={emptyForm}>
```

Comment:
Tämä koko haku kannattanee laittaa form-objektin sisään ja käyttää näitä funkkareita formin omien eventtien läpi. Näissä buttoneissa ei siis tarvitse on:clickillä kiinnittää toiminnallisuutta vaan antaa niille type="submit" ja type="reset" attribuutit ja tehdä käpistely sitten formin on:submit ja on:reset eventeissä.

Reply:
Päivitin käyttämään formia mutta empty nappi kuitenkin käyttää empty tyypin lisäksi on:clickiä koska radiot ja checkboxit ei toimi oikein sen kanssa